cartography/intel/aws/permission_relationships.py

```diff
@@ -449,6 +534,69 @@ def is_valid_rpr(rpr: Dict) -> bool:
     return True
 
 
+def load_seed_high_principals(
+    neo4j_session: neo4j.Session,
+    seed_admin_principals: List[Dict],
+) -> None:
+    set_seed_admin_query = """
+    UNWIND $principals as principal
+    MATCH (p:AWSPrincipal{arn:principal.principal_arn})
+    SET p.is_high = True, p.high_reason = principal.high_reason
+    """
+    neo4j_session.run(
+        set_seed_admin_query,
+        principals=seed_admin_principals,
+    )
+
+
+def set_remaining_high_principals(neo4j_session: neo4j.Session, current_aws_account_id: str) -> None:
+    # Set high to true if admin is true. Other queries will override
+    # this with updated reason.
+    set_high_through_admin_query = """
+        MATCH
+        (acc:AWSAccount{id:$AccountId})-[:RESOURCE]->
+        (p:AWSPrincipal)
+        WHERE p.is_admin = True
+        SET p.is_high = True,
+        p.high_reason = p.admin_reason
+    """
+    neo4j_session.run(
+        set_high_through_admin_query,
+        AccountId=current_aws_account_id,
+    )
```

Comment:
Run this last so that the high_reason for admin is the same as admin_reason. It makes more sense to give the admin reason like AdministratorFullAccess than an arbitrary high reason.